container/status.go

```diff
@@ -0,0 +1,28 @@
+package container
+
+import "github.com/nspcc-dev/neofs-api-go/v2/status"
+
+// defined in NeoFS API protocol.
+const statusSection = 3
```

Comment:
Can we just use the constant from the generated code? Or at least make `statusSection` an alias for it, removing the need for comment.